src/db/effect_bundles_to_effects_junctions_tables_vco.tsv

```diff
@@ -74,6 +74,14 @@ vco_victory_payload_ogr_gre_2_overtyrants_reign_2	vco_ogr_greasus_ward_save	5.0
 vco_victory_payload_ogr_gre_3_drakecrush	wh3_main_effect_ogr_camp_limit	2.0	faction_to_faction_own_unseen	start_turn_completed
 vco_victory_payload_ogr_gre_3_drakecrush	wh3_main_tech_trade_bonus_per_ally	20.0	faction_to_faction_own_unseen	start_turn_completed
 vco_victory_payload_ogr_gre_3_drakecrush	wh_main_effect_character_mod_ancillary_drop	10.0	faction_to_character_own	start_turn_completed
+vco_victory_payload_ksl_ort_1_there_can_only_be_one	wh3_main_effect_ice_court_training_cost_mod	-20.0	faction_to_faction_own	start_turn_completed
+vco_victory_payload_ksl_ort_1_there_can_only_be_one	wh3_main_effect_ice_court_training_time_reduction	-2.0	faction_to_faction_own	start_turn_completed
+vco_victory_payload_ksl_ort_2_the_last_bastion	wh_main_effect_force_stat_melee_defence_siege_defence	10.0	faction_to_force_own	start_turn_completed
+vco_victory_payload_ksl_ort_2_the_last_bastion	wh2_main_effect_force_enemy_campaign_movement_range	-10.0	faction_to_region_own	start_turn_completed
+vco_victory_payload_ksl_ort_3_wololo_1	wh2_main_effect_economy_gdp_mod_trade_sea_global	30.0	faction_to_faction_own_unseen	start_turn_completed
+vco_victory_payload_ksl_ort_3_wololo_2	wh_main_effect_force_stat_melee_defence_siege_defence	6.0	faction_to_force_own	start_turn_completed
+vco_victory_payload_ksl_ort_3_wololo_3	wh3_main_effect_corruption_reduction_events	-2.0	faction_to_faction_own_unseen	start_turn_completed
+vco_victory_payload_ksl_ort_3_wololo_4	vco_ksl_ort_flagellant_unlock_dummy	1.0	faction_to_faction_own_unseen	start_turn_completed
```

Comment:
This one needs more work, including scripting and multiple tables more. Use **vco_def_morathi_slaanesh_unlock_dummy** as a reference.